clash-cores/clash-cores.cabal

```diff
@@ -74,6 +74,10 @@ library
   hs-source-dirs: src
 
   exposed-modules:
+    Clash.Cores.Floating.Xilinx
+    Clash.Cores.Floating.Xilinx.Annotations
+    Clash.Cores.Floating.Xilinx.BlackBoxes
+    Clash.Cores.Floating.Xilinx.Internal
```

Comment:
I think I'd rather see these under `Clash.Core.Xilinx.Floating` on the grounds that

  * existing cores for Lattice IP are under `Clash.Cores.LatticeSemi`
  * if we add more Xilinx IP that isn't FP-related then the Xilinx IP won't be all together, which I personally think is less intuitive for someone browsing the source

Reply:
Right! I had reasoned the other way round, we'll add Floating support to more targets, and they are put together. But looking at the prelude, we have `Clash.Intel.ClockGen` and `Clash.Xilinx.ClockGen`, so your solution has precedent!